src/spikeinterface/core/channelsaggregationrecording.py

```diff
@@ -67,31 +82,20 @@ def __init__(self, recording_list, renamed_channel_ids=None):
                                 (property_dict[prop_name], rec.get_property(prop_name))
                             )
                         except Exception as e:
-                            print(f"Skipping property '{prop_name}' for shape inconsistency")
+                            print(f"Skipping property '{prop_name}' due to shape inconsistency")
                             del property_dict[prop_name]
                             break
 
         for prop_name, prop_values in property_dict.items():
-            if prop_name == "contact_vector":
-                # remap device channel indices correctly
-                prop_values["device_channel_indices"] = np.arange(self.get_num_channels())
             self.set_property(key=prop_name, values=prop_values)
 
-        # if locations are present, check that they are all different!
-        if "location" in self.get_property_keys():
-            location_tuple = [tuple(loc) for loc in self.get_property("location")]
-            assert len(set(location_tuple)) == self.get_num_channels(), (
-                "Locations are not unique! " "Cannot aggregate recordings!"
-            )
```

Comment:
Why removing this?

Reply:
A mistake or god knows what the heck I was thinking. Reverted.